### PR TITLE
Feature/edit code snippet

### PR DESCRIPTION
**Pull request by AstDaria**

### CODE REVIEW

File: content/yaml-quick-start/building-a-native-ios-app.md [hunk 1]
```diff
@@ -29,11 +29,11 @@ Add the following scripts to your `codemagic.yaml` file in order to prepare the
 In this step you can also define the build artifacts you are interested in. These files will be available for download when the build finishes. For more information about artifacts, see [here](../yaml/yaml-getting-started/#artifacts).
 
 
-{{< highlight yaml "style=paraiso-dark">}}
+{{< highlight-editable yaml "style=paraiso-dark">}}
 ios-native:
   environment:
     vars:
-      BUNDLE_ID: "io.codemagic.sample.iosnative"
+      BUNDLE_ID: $$$codemagic-sample$$$"io.codemagic.sample.iosnative"$$$
```
Comment (mikhail-tokarev) [inline content/yaml-quick-start/building-a-native-ios-app.md:36]: I think it should be `bundle-id`

File: layouts/shortcodes/highlight-editable.html [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{{ $language := .Get 0 }}
+{{ $styling := .Get 1 }}
+
+{{- ((highlight .Inner $language $styling) | replaceRE `\$\$\$(.*)\$\$\$(.*)\$\$\$` `<span editable-id="${1}" class="editable-field" data-js-editable-field>${2}</span>` | safeHTML ) -}}
```
Comment (mikhail-tokarev) [inline layouts/shortcodes/highlight-editable.html:4]: can we apply it for regular `highlight`? in this case no need to rename and think about `highlight` vs `highlight-editable`

**Review by mikhail-tokarev — COMMENTED** [review]
left couple comments

File: layouts/shortcodes/highlight-editable.html (hunk 2 above)
Comment (AstDaria) [reply layouts/shortcodes/highlight-editable.html:4]: Unfortunately we can't do that. The highlight in this case is built-in.

Comment (mikhail-tokarev) [reply layouts/shortcodes/highlight-editable.html:4]: ah ok

**Review by mikhail-tokarev — COMMENTED** [review]
🚀 great job

**Review by mikhail-tokarev — APPROVED** [review]

**Pull request merged**